### ISSUE TITLE
API proposal: force_disconnect() method to disconnect a client from a session

### ISSUE THREAD

**Issue by normanargueta** [opentok/Opentok-Python-SDK#131]
This is a proposal to add a new OpenTok Python SDK method to disconnect a client from an OpenTok session.

### opentok.force_disconnect()

The platform allows to disconnect a client connection from a session. See [docs](https://tokbox.com/developer/rest/#forceDisconnect).

### Usage

``` python
session_id = 'SESSIONID'
connection_id = 'CONNECTIONID'

# To send a request to disconnect a client:
opentok.force_disconnect(session_id, connection_id)
```

**Comment by msach22** [comment]
@normanargueta I think you added this by accident:
https://cl.ly/3b2J011S3t0j

**Comment by normanargueta** [comment]
@msach22 yes, sorry about that. I already updated the comment.

**Comment by msach22** [comment]
No problem, thanks for updating 😄

**Comment by msach22** [comment]
This has been released with 2.7.0
